src/CMakeLists.txt

```diff
@@ -10,6 +10,10 @@ if(NATS_BUILD_WITH_TLS)
   set(NATS_OPENSSL_LIBS "${OPENSSL_LIBRARIES}")
 endif(NATS_BUILD_WITH_TLS)
 
+if(NATS_BUILD_WITH_WOLFSSL)
```

Comment:
But then just above we possibly include OpenSSL libraries, is that the right thing to do?